kubedriver/location/deployment_location.py

```diff
@@ -30,7 +30,8 @@ class KubeDeploymentLocation(KubeDeploymentLocationBase):
     HELM_TLS_CA_CERT_PROP = 'helm.tls.cacert'
     HELM_TLS_CA_CERT_ALT2_PROP = 'helm.tls.ca_cert'
     HELM_TLS_CERT_PROP = 'helm.tls.cert'
-    HELM_TLS_KEY_PROP = 'helm.tls.key'
+    HELM_TLS_KEY_PROP = 'helm.tls.key' 
+    TILLER_NAMESPACE_PROP = 'tillerNamespace'
```

Comment:
In the driver we support 2 forms for each property, one with camelCase and one with snake_case. So we need a snake_case version of this property: `tiller_namespace`